packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1177,19 +1116,17 @@ new PrismaClient({
           })})`,
         )
         debug(`Generated request:`)
-        debug(query + '\n')
+        debug(message.toDebugString() + '\n')
       }
 
       await lock /** @see {@link getLockCountPromise} */
 
       return this._fetcher.request({
-        document,
+        protocolMessage: message,
+        modelName: model,
```

Comment:
Not sure if the rename makes sense for the request handler too

Reply:
There, `model` name comes from middleware parameters, so I am not quite sure we can rename it that easily